rakudo-star/content.md

```diff
@@ -0,0 +1,35 @@
+# What is Rakudo Star?
+
+Rakudo Star is a Perl 6 distribution designed for use by early adopters of the language.  It includes
+a virtual machine (the JVM or MoarVM), the Rakudo Perl 6 compiler, and a suite of modules that users
+may find useful.  This image includes the MoarVM backend for the compiler.
+
+Project homepage: http://rakudo.org
```

Comment:
The Hub's rendering engine won't linkify these, unfortunately. :cry: (gotta be explicit `[text](url)` references)